pkg/upgrade/upgradecluster/tenant_cluster.go

```diff
@@ -232,7 +233,7 @@ func (t *TenantCluster) ForEveryNodeOrServer(
 			var conn *grpc.ClientConn
 			retryOpts := retry.Options{
 				InitialBackoff: 0,
-				MaxRetries:     2,
+				MaxRetries:     100,
 				MaxBackoff:     10 * time.Millisecond,
```

Comment:
What do you think about maybe 20 as the max retries but then push the MaxBackoff to 100ms?

Reply:
Yeah, that makes more sense.  I've also increased the initial backoff to 1 millisecond, to match other uses of retry.Options.